Cheetah/testing/all_partials_tested.py

```diff
@@ -84,7 +85,7 @@ def get_partial_methods(template_packages):
 def is_partial_test_cls(cls):
     return (
         issubclass(cls, PartialTemplateTestCase) and
-        cls is not PartialTemplateTestCase and
+        cls not in _KNOWN_BASE_TESTS and
         cls.partial is not None and
         cls.method is not None
     )
```

Comment:
Figure out some duck-typing way to do this that you're happy with.

How about `and cls.__dict__.get('__test__', True)`?